packages/tocco-ui/src/Menu/ButtonMenu.js

```diff
@@ -84,7 +105,11 @@ ButtonMenu.propTypes = {
   /**
    * cypress selector string
    */
-  'data-cy': PropTypes.string
+  'data-cy': PropTypes.string,
+  /**
+   * Icon Type to be shown on the button
+   */
+  'iconType': PropTypes.string
```

Comment:
please move the icon mapping outside this tocco ui component and just add a icon property